project.sh

```diff
@@ -53,7 +53,7 @@ script-invoke () {
 		usage
 		exit 1
 	elif [[ $(grep "^$1\ (" "$script_name") ]];then
-		eval $@
+		"$@"
```

Comment:
Eval was causing string quotation issue.
Bash can evaluate strings. It takes the first part as the command